packages/mandelbrot/views/partials/navigation/components.nunj

```diff
@@ -1,5 +1,13 @@
 {% if frctl.components.size %}
```

Comment:
This condition is not really necessary anymore as we are checking that a collection includes entities before displaying it